docs/archetypes/machinelearning.md

```diff
@@ -153,6 +154,18 @@ Temporary passwords are autogenerated, and connection strings are automatically
 
 Azure Policy will enable diagnostic settings for all PaaS components in the machine learning archetype and the logs will be sent to the centralized log analytics workspace.  These policies are configured at the management group scope and are not explicitly deployed.
 
+## AKS Egress/Firewall configuration
+
+Since all traffic is redirected through the NVA / Frirewall, the following destination endpoints should be allowed on it for the AKS cluster to be properly provisionned and operational
```

Comment:
Typos to fix "Frirewall" and "provisionned".  Please change to the following:

Since all traffic is redirected through the NVA / Firewall, the following destination endpoints should be allowed on it for the AKS cluster to be properly provisioned and operational: